src/OpenColorIO/transforms/FileTransform.cpp

```diff
@@ -522,6 +527,32 @@ const char * FormatRegistry::getFormatExtensionByIndex(int capability, int index
     return "";
 }
 
+bool FormatRegistry::isFormatExtensionSupported(const char * extension) const
+{
+    // Early return false with an input of just the dot.
```

Comment:
Suggest to also protect against invalid pointer:

```
if (!extension || !*extension)
{
    return false;
}
```

Reply:
I have added those conditions for early return.

Reply:
I have also reverted all the whitespaces removed as part of my initial commits.